test/update-db.test.js

```diff
@@ -133,3 +133,56 @@ if (!NODE_8 && (isInstalled('pnpm') || process.env.CI)) {
     expect(lock).toContain(`/caniuse-lite/${ caniuse.version }:`)
   })
 }
+
+it('shows target browser changes', async () => {
+  let dir = await chdir('browserslist-diff',
+    'package.json', 'package-lock.json')
+
+  expect(runUpdate()).toEqual(
+    'Current version: 1.0.30001030\n' +
+    `New version: ${ caniuse.version }\n` +
+    'Removing old caniuse-lite from lock file\n' +
+    'Installing new caniuse-lite version\n' +
+    '$ npm install caniuse-lite\n' +
+    'Cleaning package.json dependencies from caniuse-lite\n' +
+    '$ npm uninstall caniuse-lite\n' +
+    'caniuse-lite has been successfully updated\n' +
+    '\n' +
+    'Target browser changes:\n' +
+    '- chrome 83\n' +
```

Comment:
Vary flaky test, since we can't pin desired can i use version.
We can definitely check removed browsers versions.
For added one maybe some specific browserslist query can be used to minimize changes

Reply:
Problems started earlier than i thought.
node environment suddenly has fewer updates
![Screenshot from 2020-11-08 00-30-08](https://user-images.githubusercontent.com/15820496/98451744-c6708180-2159-11eb-8237-a08d378baa5b.png)

Reply:
You can use `expect(…).toMatch(/…/)`

Reply:
As far as I understand node is an environment with he latest node version (15.1), and I still doesn't understand why it produces different results.
Running `npx browserslist` gives us the list using currently installed `caniuse-lite`, so the version of `caniuse-lite` is somehow different in different environments?

Reply:
I simplified the test to check presence of correctly formatted browser changes, since we can't tell what browser changes will pesent.